main.go

```diff
@@ -76,10 +76,10 @@ func main() {
 
 	// Get config from environment
 	config := &Config{}
-	if err := envconfig.Usage("registry-memory", config); err != nil {
```

Comment:
Interesting.. it is working with k8s https://github.com/networkservicemesh/integration-k8s-kind/blob/master/deployments/registry-memory.yaml#L22, but looks like it will not work in external case :)